app/uk/gov/hmrc/exports/migrations/repositories/LockRefreshChecker.scala

```diff
@@ -16,14 +16,13 @@
 
 package uk.gov.hmrc.exports.migrations.repositories
 
-import java.util.Date
-
 import uk.gov.hmrc.exports.migrations.LockManager.LockRefreshMarginMillis
 import uk.gov.hmrc.exports.migrations.TimeUtils
 
+import java.time.Instant
+
 class LockRefreshChecker(private val timeUtils: TimeUtils) {
-  def needsRefreshLock(lockExpiresAt: Date): Boolean =
-    lockExpiresAt == null ||
```

Comment:
New version is missing this null check, any reason to remove it?

Reply:
Did I (maybe) misinterpreted the reason why lockExpiresAt could be `null`? I mean... there is an actual chance to call this method passing `null` as parameter?